package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@extism/extism",
-  "version": "1.0.0-rc3",
+  "version": "1.0.0-rc4",
```

Comment:
Since this value is controlled by CI, I'm tempted to set this to `0.0.0-replaced-by-ci`, like the [python-sdk](https://github.com/extism/python-sdk/blob/main/pyproject.toml#L3).

Reply:
Ah, yeah, I just checked the behavior of `npm version`; it fails if we try to set the version to an existing version – we'll want to set this to `0.0.0-replaced-by-ci`.